resources/networks/hello/network.yaml

```diff
@@ -0,0 +1,87 @@
+nodes:
```

Comment:
Any reason to put this network here instead of `/test/data` ? This location will make it a "default" network that gets copied into all new warnets. Maybe the idea is to have a plugin template as a default? Either way I prefer networks that are run by tests to live in `/test/data`.

Especially because this network.yaml generates tons of LN activity between two specific nodes, not like general network activity. It seems less default-y

Reply:
I like having the 'hello' network as as something that users get when they `init` a new warnet user directory because I think it's helpful to have an "all in" example for users to look at.

That said, I can understand creating a copy of the hello network in the testing directory so that all the test stuff is isolated.